examples/web-ui/material/menus/menuDesktop.smash

```diff
@@ -0,0 +1,9 @@
+* Set Desktop
+    * Click first level page button {{page}}
+        Log "Desktop click"
+            Click '{{page}}Button'
```

Comment:
This is totally valid code, but for an example, I kind of want to avoid using this pattern. If you're clicking something, either click a full variable or a full string.